api/test/unit/views/test_media_views.py

```diff
@@ -42,6 +44,36 @@ def test_retrieve_query_count(api_client, media_type_config):
     assert res.status_code == 200
 
 
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    "path, expected_params",
+    [
+        pytest.param("tag/cat/", {"tag": "cat"}, id="tag"),
+        pytest.param("source/flickr/", {"source": "flickr"}, id="source"),
+        pytest.param(
+            "source/flickr/creator/cat/",
+            {"source": "flickr", "creator": "cat"},
+            id="source_creator",
+        ),
+    ],
+)
+def test_collection_parameters(path, expected_params, api_client):
```

Comment:
This is a good test, I wonder if we could convert it into an integration test instead of using mocks, the same we do for the other search endpoints. Here's an example from a recent PR by @dhruvkb that added such a test: https://github.com/WordPress/openverse/pull/3013/files#diff-b8ef7387c8d809dde36c541047533b8c3f1e23ccaa1455ab98de1428f0518143R158-R180

Reply:
The integration tests were added.